src/main/kotlin/onm/configuration/ConfigurationDataClass.kt

```diff
@@ -0,0 +1,52 @@
+package onm.configuration
+
+/**
+ * Data class used as configuration part for out framework.
+ * */
+data class ConfigurationDataClass(
+        /**
+         * structure of this map is like that:
+         *
+         * ```RoomType: DeviceType - count```
+         *
+         * so example:
+         *
+         * ```Kitchen : Fridge : 1```
+         * */
+        val roomsAndDevices: Map<RoomConfig, Collection<DeviceConfig>> //todo add more config options
+)
+
+/**
+ * Represents room config.
+ * */
+data class RoomConfig(
+        val roomType: RoomType,
+        val description: String,
+        val floor: Int
+)
+
+/**
+ * Represents device config.
+ * */
+data class DeviceConfig(
+        val deviceType: DeviceType,
+        val powerConsumption: PowerConsumption
+)
+
+/**
+ * Data class which represents power consumptions in different devices states.
+ * */
+data class PowerConsumption(
+        val idleState: Int? = null,
+        val workingState: Int? = null,
+        val turnedOffState: Int? = null
+) {
```

Comment:
this is nullable because this will be parsed from JSON, what if user want to use default parameters? -> he will let these fields null and we will set power consumption while instancing new device from default values -> which are defined in the enum `DeviceType`